data-protocols/ids/ids-transform-v1/src/main/java/org/eclipse/dataspaceconnector/ids/transform/type/asset/TransformKeys.java

```diff
@@ -9,18 +9,18 @@
  *
  *  Contributors:
  *       Daimler TSS GmbH - Initial implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - refactoring
  *
  */
 
-package org.eclipse.dataspaceconnector.ids.spi.transform;
+package org.eclipse.dataspaceconnector.ids.transform.type.asset;
 
 /**
  * IDS Transform Keys from the {@link org.eclipse.dataspaceconnector.spi.types.domain.asset.Asset} properties.
  */
 public class TransformKeys {
-
-    public static final String KEY_ASSET_FILE_NAME = "ids:fileName";
-    public static final String KEY_ASSET_BYTE_SIZE = "ids:byteSize";
-    public static final String KEY_ASSET_FILE_EXTENSION = "ids:fileExtension";
+    public static final String KEY_ASSET_FILE_NAME = "fileName";
```

Comment:
I thought that the asset properties should have been in the `asset:prop:<name>` format